cinema/serializers.py

```diff
@@ -1,24 +1,27 @@
 from rest_framework import serializers
 
-from cinema.models import Movie
+from cinema.models import Movie, Actor, Genre, CinemaHall
 
 
-class MovieSerializer(serializers.Serializer):
-    id = serializers.IntegerField(read_only=True)
-    title = serializers.CharField(max_length=255)
-    description = serializers.CharField()
-    duration = serializers.IntegerField()
+class MovieSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Movie
+        fields = ["id", "title", "description", "duration"]
```

Comment:
Consider adding `actors` and `genres` to the `fields` list in `MovieSerializer` to fully represent the `Movie` model. This will allow the serializer to include these many-to-many relationships in the serialized output.